content/en/blog/2021/contributor-story.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Announcing the Kubernetes Contributor Stories Series"
+linkTitle: "Share your Kubernetes Contributor Story"
+date: 2021-04-11
```

Comment:
```suggestion
date: 2021-04-21
```
I just randomly picked the date, but it needs to be bumped to match when this is intended to be released.